### PR TITLE
Upgrade React and clean up example slightly

### PR DESCRIPTION
**Pull request by hojberg**
## Problem
React is out of date and the example is a bit of a mess.

## Solution
Upgrade React and the peerDependency requirement as well as clean up the example a little (it could use a lot more help).

## Caveats/Notes
I might move `react-dom` into a sub project in the examples folder though it seems useful for tests. @newyankeecodeshop whats your thoughts on that?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,28 +7,30 @@
...
-    "build-example": "tsc; browserify examples/LoginForm/Main.js -o examples/LoginForm/public/bundle.js -t [ babelify --presets [ @babel/preset-env @babel/preset-react ] ]",
-    "test": "jest",
+    "build-example": "tsc; browserify examples/LoginForm/Main.js -o examples/LoginForm/bundle.js -t [ babelify --presets [ @babel/preset-env @babel/preset-react ] ]",
+    "test": "rm -rf dist; rm -rf build; jest",
     "tsc": "tsc",
     "prepublishOnly": "rm -rf dist; tsc --outDir dist/ --module commonjs"
   },
   "peerDependencies": {
-    "react": ">=15.0.0"
+    "react": ">=16.0.0"
```
Comment (hojberg) [inline package.json:16]: @newyankeecodeshop What version of React is the Point of Sale on?

File: package.json [hunk 2]
```diff
@@ -7,28 +7,30 @@
   "author": "Simon Højberg <r.hackr@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build-example": "tsc; browserify examples/LoginForm/Main.js -o examples/LoginForm/public/bundle.js -t [ babelify --presets [ @babel/preset-env @babel/preset-react ] ]",
-    "test": "jest",
+    "build-example": "tsc; browserify examples/LoginForm/Main.js -o examples/LoginForm/bundle.js -t [ babelify --presets [ @babel/preset-env @babel/preset-react ] ]",
+    "test": "rm -rf dist; rm -rf build; jest",
```
Comment (hojberg) [inline package.json:11]: I need to get this build folder etc out of the main package and into the examples folder at some point.

File: package.json (hunk 1 above)
Comment (newyankeecodeshop) [reply package.json:16]: @hojberg We are on React 16.8.6, which is the version that React Native 0.60 embeds.

**Review by newyankeecodeshop — APPROVED** [review]

**Comment by newyankeecodeshop** [comment]
I think having `react-dom` as a dev dependency is okay. The test renderer doesn't need it though does it?

File: package.json (hunk 1 above)
Comment (hojberg) [reply package.json:16]: Perfect. I'm prolly gonna do a major release today for that requirement, but I think it should be safe for Upserve to upgrade.

**Comment by hojberg** [comment]
@newyankeecodeshop yeah I was thinking if we needed to do a more guided component test. You're prolly right.

**Pull request merged**